03 Writing Algorithms/12 Universes/03 Equity/03 ETF Constituents Universes/99 Examples.html

```diff
@@ -1,65 +1,226 @@
+<h4>Example 1: Arbitration Of Sparse-Replicated Portfolio of SPY</h4>
+<p>The following example selects the top 100 weighted constituents in SPY and buy them according to their constituted weights in SPY, while sell the SPY in equal size for a dollar-neutral portfolio. It is believed that the top 100 have better return than the rest due to their high popularity, such that extra alpha can be obtained from active selection and management fee decay from SPY.</p>
+<div class="section-example-container">
+<pre class="csharp">public class ETFConstituentsUniverseAlgorithm : QCAlgorithm
+{
+    private Dictionary&lt;Symbol, decimal&gt; _weightBySymbol = new Dictionary&lt;Symbol, decimal&gt;();
+    private List&lt;Symbol&gt; _universe;
+
+    public override void Initialize()
+    {
+        SetCash(100000);
+        SetStartDate(2023, 6, 1);
+        // Asynchronous can utilize computational resources more efficiently.
+        UniverseSettings.Asynchronous = true;
```

Comment:
We can't use `UniverseSettings.Asynchronous = true;` when we use a class variable `_weightBySymbol`.